kolibri/core/assets/src/core-global.styl

```diff
@@ -19,19 +19,83 @@
 
 @require '~core-theme.styl'
 
+
+// http://www.paulirish.com/2012/box-sizing-border-box-ftw/
+*, *:before, *:after
+  box-sizing: inherit
+
+
+html
+  // https://davidwalsh.name/html5-boilerplate
+  font-size: 100%
+  text-size-adjust: 100%
+  overflow-y: scroll
+  box-sizing: border-box
+
+
+
 html, body
   font-family: $core-font
   color: $core-text-default
   background-color: $core-bg-canvas
 
+
 a
   color: $core-action-normal
   text-decoration: none
+  transition: color $core-time ease-out
 
   &:hover
     color: $core-action-dark
 
 
+// https://davidwalsh.name/html5-boilerplate
+img
+  -ms-interpolation-mode: bicubic
+
+
+// https://davidwalsh.name/html5-boilerplate
+svg:not(:root)
+  overflow: hidden
+
+
+button
+  // structure (from https://github.com/yahoo/pure/blob/master/src/buttons/css/buttons-core.css)
```

Comment:
(I had also had trouble getting the button to look good - still not too happy with it)